website/docs/components/table/partials/code/component-api.md

```diff
@@ -1,49 +1,107 @@
+### Table
 
 <Doc::ComponentApi as |C|>
   <C.Property @name="<:head>" @type="named block">
-    This is a named block where the content for the table head (`<thead>`) is rendered.
+    This is a named block where the content for the table head (`<thead>`) is rendered. It is unlikely that most consumers will need to use this named block directly for most use cases.
   </C.Property>
   <C.Property @name="<:body>" @type="named block">
     This is a named block where the content for the table body (`<tbody>`) is rendered.
   </C.Property>
   <C.Property @name="model" @type="array">
-    If defined, sets the data source that gets yielded by the `:body` named block.
+    Indicates the data model to be used by the table.
   </C.Property>
   <C.Property @name="columns" @type="array">
-    Use an `array` hash to define your table columns. While `key` and `label` are required, other options include `isSortable`, `align` (for text-alignment), and `width`.
+  Use a `hash` within the array to define each column. `Key` and `label` values are both required. Add `isSortable` if you would like the column to be sortable.
   </C.Property>
   <C.Property @name="sortBy" @type="string">
-    If defined, indicates which column should be pre-sorted when the table is rendered.
+    If defined, the value should be set to the key of the column that should be pre-sorted when the table is rendered.
   </C.Property>
   <C.Property @name="sortOrder" @type="string" @values={{array "asc" "desc" }} @default="asc">
     Use in conjunction with `sortBy`. If defined, indicates which direction the column should be pre-sorted in. If not defined, `asc` is applied by default.
   </C.Property>
-  <C.Property @name="onSort" @type="function">
-    Callback function invoked (if provided) when one of the sortable table headers is clicked, and a sorting is triggered. The function receives the values of `sortBy` and `sortOrder` as arguments.
-  </C.Property>
   <C.Property @name="isStriped" @type="boolean" @values={{array "false" "true" }} @default="false">
     Define on the table invocation. If set to `true`, row striping ("zebra striping") will be applied to the table.
   </C.Property>
-  <C.Property @name="isFixed" @type="boolean" @values={{array "false" "true" }} @default="false">
+  <C.Property @name="isFixedLayout" @type="boolean" @values={{array "false" "true" }} @default="false">
     If set to `true`, the `table-display`(CSS) property will be set to `fixed`, which will automatically distribute columns equally based on the total width of the table.
   </C.Property>
   <C.Property @name="density" @type="enum" @values={{array "short" "medium" "tall" }} @default="medium">
     If set, determines the density, or height, of the table’s rows.
   </C.Property>
+  <C.Property @name="align" @type="enum" @values={{array "left" "center" "right" }} @default="left">
+    If set, determines the text alignment for all cell (`td`) content in a table.
```

Comment:
should we also specify that doesn't apply to table headers as the one below?

(in the table body the `align=@align` is not declared on the `Th`, don't know it it's intentional or if it's a slip and should be there)

Reply:
After thinking about this, I think it's different depending on how you use it, I guess. When you are invoking it in a table context, it's setting it for the table. So I don't think we need to say more than that.

Reply:
re: `align` it's kind of strange because the showcase is correct: 

![CleanShot 2023-02-23 at 10 03 25](https://user-images.githubusercontent.com/4587451/220962844-ea6f4497-257c-4615-a727-cbe4207cd2e4.png)

I guess I need to figure out how it's getting that if it wasn't added to the component properly?